### ISSUE TITLE
Duplicate suggestion in auto-suggest component

### ISSUE THREAD

**Issue by holwech** [matteodem/meteor-easy-search#416]
I'm getting duplicates when using the auto-suggest component.

![autosuggest](https://cloud.githubusercontent.com/assets/7474295/12152748/80eb9d6c-b4b6-11e5-90a1-6d20c60e918e.gif)

client
```javascript
Template.tools.helpers({
	commentsIndex: function() {
		return CommentsIndex;
	}
});
```

```html
<template name='tools'>
	<div class='row'>
		<div class="autosuggest-component">
			{{> EasySearch.Autosuggest index=commentsIndex}}
		</div>
	</div>
</template>
```

server
```javascript
CommentsIndex = new EasySearch.Index({
	collection: Comments,
	fields: ['comment'],
	engine: new EasySearch.MongoDB()
});
```

**Comment by matteodem** [comment]
Can you provide me a reproduction repository?

**Comment by holwech** [comment]
Here you go:
- Repo https://github.com/holwech/auto-suggest
- Deployed http://auto-suggest.meteor.com/

**Comment by matteodem** [comment]
Thank you, I'll have a look now.

**Comment by matteodem** [comment]
This should be fixed in the latest release `2.0.1` of the autosuggest component
